config/default.cfg

```diff
@@ -27,10 +27,10 @@ cfg$regionmapping <- "config/regionmappingH12.csv"
 ### Additional (optional) region mapping, so that those validation data can be loaded that contain the corresponding additional regions.
 cfg$extramappings_historic <- ""
 #### Current input data revision (<mainrevision>.<subrevision>) ####
-cfg$inputRevision <- "6.79"
+cfg$inputRevision <- "6.83edgeTransportReloaded"
```

Comment:
this input data sound abit like test-data. We do not have input data with a "better" version number, right?